logzio/datasource_metrics_account_test.go

```
@@ -0,0 +1,128 @@
+package logzio
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/logzio/logzio_terraform_provider/logzio/utils"
+	"os"
+	"regexp"
+	"strconv"
+	"testing"
+)
+
+func TestAccDataSourceMetricsAccount(t *testing.T) {
+	resourceName := "logzio_metrics_account.metrics_account_datasource"
+	dataSourceName := "data.logzio_metrics_account.metrics_account_datasource_by_id"
+	accountId, _ := strconv.ParseInt(os.Getenv(envLogzioAccountId), utils.BASE_10, utils.BITSIZE_64)
+	email := os.Getenv(envLogzioEmail)
+	accountName := "test_datasource_create"
+	defer utils.SleepAfterTest()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheckApiToken(t)
+			testAccPreCheckEmail(t)
+			testAccPreCheckAccountId(t)
+		},
+		ProviderFactories: testAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMetricsAccountDataSourceResource(email, accountId, accountName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "account_name", accountName),
+					resource.TestCheckResourceAttr(resourceName, "plan_uts", "100"),
+				),
+			},
+			{
+				Config: testAccMetricsAccountDataSourceResource(email, accountId, accountName) +
+					testAccCheckLogzioMetricsAccountDatasourceConfig(),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr(dataSourceName, "account_name", accountName),
+					resource.TestCheckResourceAttr(dataSourceName, "plan_uts", "100"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccDataSourceMetricsAccountByAccountName(t *testing.T) {
+	resourceName := "logzio_metrics_account.metrics_account_datasource"
+	dataSourceName := "data.logzio_metrics_account.metrics_account_datasource_by_id"
+	accountId, _ := strconv.ParseInt(os.Getenv(envLogzioAccountId), utils.BASE_10, utils.BITSIZE_64)
```

why this error is ignored?